data/json/items/comestibles/raw_veggy.json

```
@@ -416,6 +416,23 @@
     "smoking_result": "dry_veggy",
     "vitamins": [ [ "vitC", 9 ], [ "calcium", 6 ], [ "iron", 6 ] ]
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "raw_bottle_gourd",
+    "name": { "str": "raw bottle gourd" },
+    "copy-from": "cucumber",
+    "looks_like": "cucumber",
+    "weight": "1000 g",
+    "spoils_in": "31 days",
+    "comestible_type": "FOOD",
+    "symbol": "o",
+    "quench": 2,
+    "calories": 15,
+    "description": "A vegetable from the gourd family, bottle gourd was one of the very first plants to be domesticated by early humans.  It can be further prepared to produce a hard, watertight shell, usable as a bio-degradable liquid container.  Eating it doesn't look worth it.",
```

```suggestion
    "description": "A vegetable from the gourd family, bottle gourd was one of the very first plants to be domesticated by pre-historic Native Americans.  It can be further prepared to produce a hard, watertight shell, usable as a bio-degradable liquid container.  Eating it doesn't look worth it.",
```
Early humans is not right, as that would suggest by earlier human species. The first Native American settlers cultivated gourds, they were just guys like us, only with less gourd at first. ;)